tools/experiments/experiments.js

```diff
@@ -101,6 +101,7 @@ const EXPERIMENTS = [
     id: 'no-auth-in-prerender',
     name: 'Delay amp-access auth request until doc becomes visible.',
     spec: '',
+    removalIssue: 'https://github.com/ampproject/amphtml/issues/3824',
```

Comment:
Maybe `cleanupIssue`

Please add bug or just `TODO` to every experiment and then assert below that the field is present.